samples/boards/qemu_cortex_a53/reserved_memory/dts/bindings/sample_driver.yaml

```diff
@@ -0,0 +1,18 @@
+description: Sample Driver
+
+compatible: "sample_driver"
+
+include:
+  - name: base.yaml
+    property-allowlist: ['reg', 'label']
+
+properties:
+    memory-region:
+      type: phandle
+      required: false
+      description: phandle to the reserved memory child node
+
+    linker-section:
```

Comment:
This property doesn't seem to be used so is it needed here at all?

Reply:
leftover. fixing.